### DISCUSSION TITLE
Images of the last row of a gallery are bigger than the others, but I want alle images have the same size

### DISCUSSION THREAD

**Discussion by michaelfluegge** [GetPublii/Publii#1917] [Help] [answered]
I typically use galleries with three images in a row. In case the last row contains only one or two images, these images are bigger than the other images in the rows above, as you can see in the screenshot.

How can I achieve that the images in the last row have the same size as the other images? I haven't found any hint the forum or the docs ( under https://getpublii.com/dev/how-images-work-in-publii/#gallery-images there is no switch, as I understand the code)

![Bild 28 12 24 um 17 42](https://github.com/user-attachments/assets/554ec7f1-d3dd-4cc2-a19c-68c142ffd1b7)

**Comment by bobmitro** [comment, marked as answer]
Try this one: add the following CSS rule via the Custom CSS tool: 

```
.gallery__item {
   flex-grow: 0 !important;
}
```

**Reply by michaelfluegge** [reply]
That's cool. Thanks for the quick answer.
